image-swipe.android.ts

```diff
@@ -67,6 +67,14 @@ export class ImageSwipe extends ImageSwipeBase {
         // Coerce selected index after we have set items to native view.
         pageNumberProperty.coerce(this);
     }
+
+    public nextPage(): void {
```

Comment:
For animation to occur on android shouldn't this line: https://github.com/Buuhuu/nativescript-image-swipe/blob/dd03eb2f5a2cbf59e899945f0c46d949d20c2f7e/image-swipe.android.ts#L61 be changed to `this.nativeView.setCurrentItem(value, true)` (so basically a similar approach as on iOS)?